packages/jest-changed-files/src/git.ts

```diff
@@ -72,12 +72,12 @@ const adapter: SCMAdapter = {
   },
 
   getRoot: async cwd => {
-    const options = ['rev-parse', '--show-toplevel'];
+    const options = ['rev-parse', '--show-cdup'];
 
     try {
       const result = await execa('git', options, {cwd});
 
-      return result.stdout;
+      return path.join(cwd, result.stdout);
```

Comment:
According to test failures we need to add a trailing path separator there. Can you verify if that's necessary or maybe we could just update our tests?

Reply:
seems more sensible to update the tests for that, no?

Reply:
Ok so I found that for `execa` it doesn't matter if this path separator is at the end.